src/mono/wasi/wasi.proj

```diff
@@ -20,6 +20,8 @@
     <_WasiLinkRspPath>$(NativeBinDir)src\wasi-link.rsp</_WasiLinkRspPath>
     <WasmNativeStrip Condition="'$(ContinuousIntegrationBuild)' == 'true'">false</WasmNativeStrip>
     <WasiSdkRoot Condition="'$(WasiSdkRoot)' == ''">$(WASI_SDK_PATH)</WasiSdkRoot>
+    <WASI_SDK_PATH>$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), 'wasi-sdk'))</WASI_SDK_PATH>
+    <WASI_SDK_PATH>$([MSBuild]::EnsureTrailingSlash('$(WASI_SDK_PATH)').Replace('\', '/'))</WASI_SDK_PATH>
```

Comment:
We first assign value to `WasiSdkRoot` that is `WASI_SDK_PATH` and only then manipulate with `WASI_SDK_PATH`. Shouldn't we do it in the reverse order?

Reply:
Right now I just wanted quick hack, so see how p2 would work at all. We should probably separate SDK provisioning fixes into separate PR. 

And solve https://github.com/dotnet/runtime/issues/101528 in that too.
Which amounts to updating the VM images.

Regarding `WasiSdkRoot` I agree we could unify it.